db/migrate/20190214075550_add_followup_fields_to_patient.rb

```diff
@@ -0,0 +1,6 @@
+class AddFollowupFieldsToPatient < ActiveRecord::Migration[5.1]
+  def change
+    add_column :patients, :contacted_by_counsellor, :boolean, default: false
```

Comment:
This is better tracked by creating records of the communication model. 
However, only a User can create a Communication record. We'll either have to make communication user polymorphic, or preferably use the new combined User and Admin model after the roles and permissions refactoring.